GatekeeperStudyHall/Assets/ScriptableObjects/TraitHandlerSO.cs

```diff
@@ -23,26 +25,45 @@ public enum Trait
 }
 
 
+/// <summary>
+/// Has methods that perform the traits of given players at given rolls.
+/// </summary>
 [CreateAssetMenu(fileName = "New_TraitHandlerSO", menuName = "Scriptable Objects/TraitHandlerSO")]
-public class TraitHandlerSO : ScriptableObject
+public class TraitHandlerSO : ScriptableObject // why not make this a MonoBehavior?
```

Comment:
I remember at some point we discovered ScriptableObjects and started to implement them in any place they seemed possible. We currently use TraitHandlerSO's in DiceRoll.cs, States.cs, and TraitDebugger.cs. I think we decided that instead of having a game object to hold the TraitHandler script we would just instantiate the TraitHandlerSO in the scripts that need it.